system-tests/resources/cli-tools/entrypoint.sh

```diff
@@ -68,7 +61,22 @@ for participant in "${PARTICIPANTS[@]}"; do
 
   # Wait for participant registration.
   ./validate_onboarding.sh "$participantDid" "$cmd"
-done
+}
+
+# Read participants from participants.json file.
+participants=$(jq -r '.include | map([.participant, .region])[] | @sh' /common-resources/participants.json)
```

Comment:
Great query! However, not super easy to read.

nit: maybe worth adding an additional comment? This way it would be easier to understand the following code.

E.g.:

```suggestion
# Read participants from participants.json file. $participants will contain participants and regions 
# in a shell readable format as e.g.: 'company1' 'eu' 'company2' 'eu' 'company3' 'us'
participants=$(jq -r '.include | map([.participant, .region])[] | @sh' /common-resources/participants.json)
```